Buildings/Experimental/DHC/CentralPlants/Cooling/Plant.mo

```diff
@@ -0,0 +1,377 @@
+    redeclare final package Medium=Medium)
+    "Chilled water bypass valve mass flow meter"
+    annotation (Placement(transformation(extent={{-10,-10},{10,10}},rotation=180,origin={30,-90})));
+  Buildings.Fluid.Sensors.TemperatureTwoPort senTCHWSup(
+    redeclare final package Medium=Medium,
+    final m_flow_nominal=mCHW_flow_nominal)
+    "Chilled water supply temperature"
+    annotation (Placement(transformation(extent={{-10,10},{10,-10}},rotation=0,origin={210,-90})));
+  Buildings.Experimental.DHC.CentralPlants.Cooling.Controls.ChilledWaterPumpSpeed chiWatPumCon(
+    tWai=0,
+    final m_flow_nominal=mCHW_flow_nominal,
+    final dpSetPoi=dpSetPoi)
+    "Chilled water pump controller"
+    annotation (Placement(transformation(extent={{-140,-12},{-120,8}})));
+  Buildings.Experimental.DHC.CentralPlants.Cooling.Controls.ChillerStage chiStaCon(
+    final tWai=tWai,
+    final QChi_nominal=QChi_nominal)
+    "Chiller staging controller"
+    annotation (Placement(transformation(extent={{-240,184},{-220,204}})));
+  Modelica.Blocks.Sources.RealExpression mPum_flow(
+    final y=pumCHW.port_a.m_flow)
+    "Total chilled water pump mass flow rate"
+    annotation (Placement(transformation(extent={{-210,-8},{-190,12}})));
+  Buildings.Fluid.Sensors.TemperatureTwoPort senTCHWRet(
+    redeclare final package Medium=Medium,
+    final m_flow_nominal=mCHW_flow_nominal)
+    "Chilled water return temperature"
+    annotation (Placement(transformation(extent={{-10,-10},{10,10}},rotation=0,origin={-240,-90})));
+  Buildings.Fluid.Sources.Boundary_pT expTanCW(
+    redeclare final package Medium=Medium,
+    nPorts=1)
+    "Condenser water expansion tank"
+    annotation (Placement(transformation(extent={{40,110},{20,130}})));
+  Buildings.Fluid.Sources.Boundary_pT expTanCHW(
+    redeclare final package Medium=Medium,
+    nPorts=1)
+    "Chilled water expansion tank"
+    annotation (Placement(transformation(extent={{-110,-40},{-90,-20}})));
+  Buildings.Fluid.Sensors.MassFlowRate senMasFlo(
+    redeclare final package Medium=Medium)
+    "Chilled water return mass flow"
+    annotation (Placement(transformation(extent={{-190,-100},{-170,-80}})));
+  Buildings.Controls.OBC.CDL.Conversions.BooleanToReal chiOn[numChi]
+    "Convert chiller on signal from boolean to real"
+    annotation (Placement(transformation(extent={{-40,184},{-20,204}})));
+  Buildings.Controls.OBC.CDL.Continuous.MultiSum totPPum(
+    nin=4)
+    "Total pump power"
+    annotation (Placement(transformation(extent={{260,150},{280,170}})));
+  Buildings.Controls.OBC.CDL.Continuous.MultiSum totPFan(
+    nin=2)
+    "Total fan power"
+    annotation (Placement(transformation(extent={{260,190},{280,210}})));
+  Buildings.Controls.OBC.CDL.Continuous.MultiSum totPCoo(
+    nin=2)
+    "Total cooling power"
+    annotation (Placement(transformation(extent={{260,230},{280,250}})));
+  Buildings.Experimental.DHC.EnergyTransferStations.BaseClasses.Junction joiCHWRet(
```

Comment:
Set `energyDynamics=SteadyState` (but not final)
See conversation [here](https://github.com/lbl-srg/modelica-buildings/pull/2474) for more details.